tests/integration/models/reddit/test_subreddit.py

```diff
@@ -716,6 +717,14 @@ def test_update(self, _):
             assert before_settings == after_settings
 
 
+class TestSubredditModmail(IntegrationTest):
+    def test_call__conversation_id(self):
+        conversation_id = 'ik72'
+        conversation = self.reddit.subreddit('all').modmail(
```

Comment:
This is lazy, so it doesn't actually fetch data. While the data isn't in the format we will inevitably want it to be in, we should verify that the request happens successfully and prepare the cassette for it. The nice thing is that once the request is made and saved here, we should be able to update the code without needing to update the cassette as it'll contain all the data needed.